packages/typespec-ts/test/integration/multipart.spec.ts

```diff
@@ -82,79 +53,64 @@ describe("MultiPartClient Rest Client", () => {
         "application/octet-stream"
       );
     });
-    it("createFile could return file objects", async () => {
-      const file = createFile(await readFile(imgPath), "hello.jpg", {
-        type: "image/jpg"
-      });
-      const result = await client
-        .path("/multipart/form-data/check-filename-and-content-type")
-        .post({
-          contentType: "multipart/form-data",
-          body: {
-            id: "123",
-            profileImage: file
-          }
-        });
-      assert.strictEqual(result.status, "204");
-    });
-    it("createFileFromStream could return file objects", async () => {
-      const content = await readFile(imgPath);
-      const file = createFileFromStream(() => content, "hello.jpg", {
-        type: "image/jpg"
-      });
+    it("allows specifying MIME type and filename", async () => {
```

Comment:
just curious without the helper `createFileFromStream` how to feed the big file in a streaming way?

Reply:
You can just pass the stream directly:

```ts
const result = await client
        .path("/multipart/form-data/check-filename-and-content-type")
        .post({
          contentType: "multipart/form-data",
          body: [
            { name: "profileImage", body: fs.createReadStream("/path/to/file"), filename, contentType }
          ]
        });
```